apps/user/src/services/auth/api.ts

```diff
@@ -1,17 +1,22 @@
 import { maru } from '@/apis/instance/instance';
-import { PostLoginAuthReq, PostJoinAuthReq } from '@/types/auth/remote';
+import { authorization } from '@/apis/token';
+import { PostJoinAuthReq, PostLoginAuthReq } from '@/types/auth/remote';
 
 export const postLoginUser = async ({ email, password }: PostLoginAuthReq) => {
     const { data } = await maru.post('/auth', { email, password });
     return data;
 };
 
-export const postJoinUser = async ({ email, code, password }: PostJoinAuthReq) => {
-    const { data } = await maru.post('/user', { email, code, password });
+export const postJoinUser = async ({ email, name, code, password }: PostJoinAuthReq) => {
+    const { data } = await maru.post('/user', { email, name, code, password });
     return data;
 };
 
 export const postRequestEmail = async (email: string) => {
     const { data } = await maru.post(`/user/verification?email=${email}`);
     return data;
 };
+
+export const deleteUser = async () => {
```

Comment:
```suggestion
export const deleteLogoutUser = async () => {
```
는 어떨까요 너무 deleteUser는 계정 삭제 같아요